package.json

```diff
@@ -58,7 +58,8 @@
         "statements": 95
       }
     },
-    "collectCoverage": true
+    "collectCoverage": true,
+    "testURL": "http://localhost/"
```

Comment:
Did you have to add this to get the tests to run in your environment?

Reply:
Originally, yes. But I've just tried it again without the `testURL`, and the tests seem to run fine. Maybe I wasn't set up correctly? First time using nix...